pyproject.toml

```diff
@@ -35,6 +35,7 @@ dependencies = [
 test = [
     "coverage[toml]>=6.5",
     "pytest",
+    "pyspark",
```

Comment:
you are not allowed to add `pyspark` as dependency. use `sql_backend` fixtures.

Reply:
removed dependency @nfx